Description: Add "Chromium" and "Raspbian" to the user-agent text.

Index: chromium-browser-65.0.3325.181/content/common/user_agent.cc
===================================================================
--- chromium-browser-65.0.3325.181.orig/content/common/user_agent.cc
+++ chromium-browser-65.0.3325.181/content/common/user_agent.cc
@@ -177,13 +177,21 @@ std::string BuildUserAgentFromOSAndProdu
   // Derived from Safari's UA string.
   // This is done to expose our product name in a manner that is maximally
   // compatible with Safari, we hope!!
+
+  // We want to also add Chromium
+  std::string realproduct = product.c_str();
+  size_t pos;
+  if ((pos = realproduct.find("/")) != std::string::npos)
+      realproduct.replace(0, pos, "Chromium");
+
   std::string user_agent;
   base::StringAppendF(
       &user_agent,
-      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) %s Safari/%d.%d",
+      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) Raspbian %s %s Safari/%d.%d",
       os_info.c_str(),
       WEBKIT_VERSION_MAJOR,
       WEBKIT_VERSION_MINOR,
+      realproduct.c_str(),
       product.c_str(),
       WEBKIT_VERSION_MAJOR,
       WEBKIT_VERSION_MINOR);
